src/librbd/ExclusiveLock.h

```diff
@@ -17,7 +17,7 @@ class ExclusiveLock : public ManagedLock<ImageCtxT> {
   }
 
   ExclusiveLock(ImageCtxT &image_ctx);
-
+  ~ExclusiveLock() = default;
```

Comment:
Nit: base class [1] has a virtual destructor, so by definition this already has a virtual destructor

[1] https://github.com/adamemerson/ceph/blob/1dc449ac5d8ed087af677b61d6da94113bbdf7e4/src/librbd/ManagedLock.h#L47